src-tauri/src/mod_management/mod.rs

```diff
@@ -31,3 +35,96 @@ pub fn set_mod_enabled_status(
 
     Ok(())
 }
+
+
+/// Parses `mod.json` for mod name
+// TODO: Maybe pass PathBuf or serde json object
+fn parse_mod_json_for_mod_name(mod_json_path: String) -> Result<String, anyhow::Error> {
+    // Read file into string and parse it
+    let data = std::fs::read_to_string(mod_json_path)?;
+    let parsed_json: serde_json::Value = serde_json::from_str(&data)?;
+
+    // Extract mod name
+    let mod_name = match parsed_json.get("Name").and_then(|value| value.as_str()) {
```

Comment:
We should enforce somewhere (Northstar-side) what is a correct mod format; I fear the `"name"` instead of `"Name"` issue here.

Reply:
This seems to be enforced to `"Name"` based on NorthstarLauncher source code: https://github.com/R2Northstar/NorthstarLauncher/blob/6f46d0ec44e95252b2e4fa08bd48290383ea1d59/NorthstarDLL/modmanager.cpp#L58

Sidenote: I'm planning to do a mods sanity checker but that's kinda outside of the scope of the PR.